Orange/widgets/classify/owrandomforest.py

```diff
@@ -6,26 +6,26 @@
 
 from Orange.data import Table
 from Orange.preprocess.preprocess import Preprocess
-from Orange.classification.random_forest import (RandomForestLearner,
+from Orange.classification.random_forest import (RandomForestClsLearner,
                                                  RandomForestClassifier)
 from Orange.widgets import widget, settings, gui
 from Orange.widgets.utils.sql import check_sql_input
 
 
 class OWRandomForest(widget.OWWidget):
-    name = "Random Forest"
+    name = "Random Forest Classification"
     description = "Random forest classication algorithm."
     icon = "icons/RandomForest.svg"
 
     inputs = [("Data", Table, "set_data"),
               ("Preprocessor", Preprocess, "set_preprocessor")]
-    outputs = [("Learner", RandomForestLearner),
+    outputs = [("Learner", RandomForestClsLearner),
                ("Classifier", RandomForestClassifier)]
 
     want_main_area = False
     resizing_enabled = False
 
-    learner_name = settings.Setting("Random Forest Learner")
+    learner_name = settings.Setting("Random Forest Cls Learner")
```

Comment:
'Cls' learner? I'd prefer 'RF Classification Learner' if there is need to abbreviate.